docs/source/data/errors.mdx

```diff
@@ -5,6 +5,8 @@ description: Making errors actionable on the client and server
 
 import TopLevelAwait from "../shared/top-level-await.mdx"
 
+> Apollo Server v4 introduced a regression where providing invalid variables incorrectly yields a 200 status code. The mitigation for this regression is to provide the `status400WithErrorsAndNoData: true` option to your `ApolloServer` constructor. More details can be found [in the migration guide](../migration/#known-regressions).
```

Comment:
```suggestion
> Apollo Server v4 introduced a regression where providing invalid variables yields a 200 status code instead of 400. To mitigate this regression, provide the `status400WithErrorsAndNoData: true` option to your `ApolloServer` constructor. For more information, see the [migration guide](../migration/#known-regressions).
```